docs/_layouts/pages.html

```diff
@@ -86,6 +86,16 @@
 </div>
 
 <div class='limiter clearfix'>
+
+  <div class="pad2y">
+    <div class="margin3 col9 keyline-all round space-bottom contain fill-orange">
+      <div class="contain pad2 fill-lighten3">
+        <h3>Legacy</h3>
+        <p class="prose">Mapbox.js is no longer in active development. To learn more about our newer mapping tools see <a href="https://docs.mapbox.com/mapbox-gl-js/">Mapbox GL JS.</a></p>
```

Comment:
LGTM. Note, https://docs.mapbox.com/mapbox-gl-js/ now resolves to https://docs.mapbox.com/mapbox-gl-js/guides/. Should we just make this the full URL? @mikelmaron